add-on/src/lib/ipfs-proxy/pre-mfs-scope.js

```diff
@@ -1,4 +1,5 @@
-const Path = require('path')
+// Use path-browserify for consistent behavior between browser and tests on Windows
```

Comment:
I think explicitly using `path-browserify` (which provides posix version by default) is bit safer than assuming browserify-ing process will swap the implementation. 

Let's merge this, and if at some point `path-browserify` turns out to be not enough, we can swap it with own implementation / wrapper.

@alanshaw  does it sound ok to you?